Design Update!

taty2010 opened the pull request:
# Summary

Made a few design and content changes to the quickstart template to make our starter template more appealing. 

## Changes
Added:
- template type title
- background gradient and color customizable blob blurbs that you can change to match the framework.
- CTA link for Deploy to Netlify and repo link
-  Brief description of the template type 

Removed:
- 'Here are some resources to help you on your Astro + Netlify journey:' text



| Before  | After |
|------------- | ------------- |
|  ![astro-quickstart-template](https://user-images.githubusercontent.com/43764894/182444028-17217ca3-a54a-42af-aa82-33a96faebbcd.jpg) | ![Screen Shot 2022-08-02 at 12 21 06 PM](https://user-images.githubusercontent.com/43764894/182444231-70208332-e49e-45f3-a28b-90261140bf8f.png) |

tzmanics commented:
This is looking awesome!!!

3 points of feedback to take or pass on:
- i think naming the new styling `base.scss` may be confusing as it seems like it's a base file of the astro base app, possibly `template-demo-styling` or just `demo-styling` or anything along those lines may be more clear.
- since it's a visual change can you add before and after screenshots, just to set the precedence from now on?
- would you want to tackle the 'README.md` section in this PR to or make a separate documentation PR?

key takeaway: NICE WORK! I'm stoked for this glow up or blob up, either way, woohooo!

taty2010 commented:
@tzmanics and @maxcell I updated the file name to 'demo-styling', added in the before and after pictures, and updated the README. 

Let me know if I should add anything else to the README. I figured I could just go over what would possibly be the most confusing part, everything else is pretty basic.

maxcell commented:
I love the design of this and I would love to see if we could play around with maybe recreating this with using CSS only. The way I would go about changing this is having a few more `<div>`s and essentially layering the SVG in the background and making it invisible to screen readers. We could then use [CSS Variables ](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties). I would be happy to pair on this with you Tatyana!

maxcell commented on src/style/demo-styling.css:
```diff
@@ -56,68 +58,77 @@ html {
 }
 
 .bttn.primary{
+  display: inline-block;
+  width: max-content;
```
I needed to add this due to the box shadow not working properly in Safari specifically. It appears that for some reason that the `inset` property only worked for block level elements. And to make sure that the button stays in one line I made the width only expand till `max-content`

taty2010 replied:
Thanks for catching that 💜

tzmanics commented on src/style/demo-styling.css:
```diff
@@ -0,0 +1,134 @@
+    background: var(--color-bg);
+    font-family: var(--main-family);
+    min-height: 100%;
+  }
```
```suggestion
}
```

tzmanics reviewed (approved):
Looks fantastic + appreciate the readme addition!
